File: answers/fizzBuzz.js
"use strict";

var fs = require("fs");

var file = fs.readFileSync(process.argv[2], "utf-8");
file
    .split("\n")
    .map((line) => {
        return line
            .replace(/\r/g, "")
            .replace(/\s*$/g, "")
            .split(" ");

    })
    .forEach((line) => {
        var output = "";
        for (var i = 1; i <= line[2]; i++) {
            if (i % line[0] === 0 && i % line[1] === 0) {
                output += "FB ";
            } else if (i % line[0] === 0) {
                output += "F ";
            } else if (i % line[1] === 0) {
                output += "B ";
            } else {
                output += i + " ";
            }
        }
        console.log(output.replace(/\s$/, ""));
    });

File: answers/python/fizzBuzz.py
# Fizzbuzz.py
import sys

with open(sys.argv[1], 'r') as file:
    for line in file:
        nums = line.split()
        i = int(nums[2])
        n = 1
        output = ""
        while n <= i:
            if  n % int(nums[0]) == 0 and n % int(nums[1]) == 0:
                output += "FB "
            elif n % int(nums[0]) == 0:
                output += "F "
            elif n % int(nums[1]) == 0:
                output += "B "
            else: 
                output += (str(n) + " ")
            n += 1
        print(output.rstrip())